molscrub/utils.py

```diff
@@ -8,27 +8,36 @@
 
 import numpy as np
 
+from .espaloma_minim import EspalomaMinimizer
+
 def optimize_conformers(mol: Mol, ff: str="mmff94", max_ff_iter: int = 400):
     optimized_energies = []
     
-    for conf_id in range(mol.GetNumConformers()):
-
-        if (ff=="mmff94" or ff=="mmff94s") and AllChem.MMFFHasAllMoleculeParams(mol):
-            ff : ForceField = AllChem.MMFFGetMoleculeForceField(mol, AllChem.MMFFGetMoleculeProperties(mol), confId=conf_id)
-        else:
-            ff : ForceField = AllChem.UFFGetMoleculeForceField(mol, confId=conf_id)
-
-        success = ff.Minimize(maxIts=max_ff_iter)
-        energy = 0.0
-        # print(f"Success: {success}")
-        # Check if minimization is successful. 
-        if success == 0:
-            energy = ff.CalcEnergy()
-            optimized_energies.append((conf_id, energy))
-        else:
-            success = ff.Minimize(maxIts=2*max_ff_iter)
-            energy = ff.CalcEnergy()
-            optimized_energies.append((conf_id, energy))
+    if (ff == "espaloma"):
+        ff = EspalomaMinimizer()
+        mol, energies = ff.minimize(mol)
+        optimized_energies = list(zip(range(len(energies)), energies))
+
+    else:
+        for conf_id in range(mol.GetNumConformers()):
+
+
+            if (ff=="mmff94" or ff=="mmff94s") and AllChem.MMFFHasAllMoleculeParams(mol):
+                ff : ForceField = AllChem.MMFFGetMoleculeForceField(mol, AllChem.MMFFGetMoleculeProperties(mol), confId=conf_id)
```

Comment:
just noticed we are not passing the variant "mmff94" or "mmff94s" to `MMFFGetMoleculeProperties`
https://www.rdkit.org/docs/source/rdkit.Chem.rdForceFieldHelpers.html#rdkit.Chem.rdForceFieldHelpers.MMFFGetMoleculeProperties